packages/turf-polygonize/test.js

```diff
@@ -0,0 +1,51 @@
+const test = require('tape'),
+  { featureCollection, lineString, polygon } = require('@turf/helpers'),
+  polygonize = require('./'),
+  fs = require('fs'),
+  path = require('path');
+
+const directories = {
+  in: path.join(__dirname, 'test', 'in') + path.sep,
+  out: path.join(__dirname, 'test', 'out') + path.sep,
+};
+
+function getFullPath(filename, type='in') {
+  return path.join(directories[type], filename);
+}
+
+function readJsonFile(filename, type='in') {
```

Comment:
You can use `load-json-file` which handles reading GeoJSON/JSON, most of the recent modules are using this.